src/hatti/views/photos.cljs

```diff
@@ -91,6 +78,11 @@
                                 js/PhotoSwipeUI_Default
                                 (clj->js photos)
                                 options)]
+    (set! (.. gallery
+              -options
+              -getImageURLForShare)
+          (fn [share-button-data]
+            (or (aget gallery "currItem" "original-src") "")))
```

Comment:
In their [docs](http://photoswipe.com/documentation/options.html), the `getImageURLForShare` function by default returns the `src` attribute of the currently selected item or an empty string. We decided to go with the same approach of returning either the `original-src` or an empty string. We are not sure what would happen if we returned null